thamos/lib.py

```diff
@@ -325,6 +326,7 @@ def advise(
         "recommendation_type": recommendation_type,
         "debug": debug,
         "force": force,
+        "is_kebechet": metadata_is_kebechet,
```

Comment:
why not simply `is_kebechet`? like for `is_s2i`

Reply:
you can directly pass the string representation of `enum`, following openapi on user-api

Reply:
Is this implemented yet?